modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -4618,32 +4357,32 @@ private void registerMetrics() {
 
         MetricRegistry reg = ctx.metric().registry(GridMetricManager.IGNITE_METRICS);
 
-        reg.register("fullVersion", this::getFullVersion, String.class, FULL_VER_DESC);
-        reg.register("copyright", this::getCopyright, String.class, COPYRIGHT_DESC);
+        reg.register("fullVersion", this::fullVersion, String.class, FULL_VER_DESC);
+        reg.register("copyright", () -> COPYRIGHT, String.class, COPYRIGHT_DESC);
 
-        reg.register("startTimestampFormatted", this::getStartTimestampFormatted, String.class,
+        reg.register("startTimestampFormatted", this::startTimeFormatted, String.class,
             START_TIMESTAMP_FORMATTED_DESC);
 
         reg.register("isRebalanceEnabled", this::isRebalanceEnabled, IS_REBALANCE_ENABLED_DESC);
-        reg.register("uptimeFormatted", this::getUpTimeFormatted, String.class, UPTIME_FORMATTED_DESC);
-        reg.register("startTimestamp", this::getStartTimestamp, START_TIMESTAMP_DESC);
-        reg.register("uptime", this::getUpTime, UPTIME_DESC);
-        reg.register("osInformation", this::getOsInformation, String.class, OS_INFO_DESC);
-        reg.register("jdkInformation", this::getJdkInformation, String.class, JDK_INFO_DESC);
-        reg.register("osUser", this::getOsUser, String.class, OS_USER_DESC);
-        reg.register("vmName", this::getVmName, String.class, VM_NAME_DESC);
-        reg.register("instanceName", this::getInstanceName, String.class, INSTANCE_NAME_DESC);
-
-        reg.register("currentCoordinatorFormatted", this::getCurrentCoordinatorFormatted, String.class,
+        reg.register("uptimeFormatted", this::upTimeFormatted, String.class, UPTIME_FORMATTED_DESC);
+        reg.register("startTimestamp", this::startTimestamp, START_TIMESTAMP_DESC);
+        reg.register("uptime", this::upTime, UPTIME_DESC);
+        reg.register("osInformation", this::osInformation, String.class, OS_INFO_DESC);
+        reg.register("jdkInformation", this::jdkInformation, String.class, JDK_INFO_DESC);
+        reg.register("osUser", this::osUser, String.class, OS_USER_DESC);
+        reg.register("vmName", this::vmName, String.class, VM_NAME_DESC);
+        reg.register("instanceName", this::name, String.class, INSTANCE_NAME_DESC);
```

Comment:
`name` -> `instanceName`

Reply:
> `name` -> `instanceName`

We shoulkd not change public api Ignite.name(). And I don't like adding one more function for the same. Makes sense?

Reply:
Ok, lets use `name()`